src/Fixer/FunctionNotation/StaticLambdaFixer.php

```diff
@@ -134,7 +134,7 @@ private function findExpressionEnd(Tokens $tokens, int $index): int
      */
     private function hasPossibleReferenceToThis(Tokens $tokens, int $startIndex, int $endIndex): bool
     {
-        for ($i = $startIndex; $i < $endIndex; ++$i) {
```

Comment:
maybe this is correct but `private function findExpressionEnd(Tokens $tokens, int $index): int` should return `$nextIndex` and not `$index` ?

Reply:
nevermind, already merged :D   :+1:

Reply:
@SpacePossum , if still relevant, do not hesitate to continue on the discussion :)